src/main/java/teammates/ui/controller/InstructorStudentRecordsAjaxPageAction.java

```diff
@@ -67,7 +73,8 @@ public ActionResult execute() throws EntityDoesNotExistException {
                       + "in course <span class=\"bold\">[" + courseId + "]</span>";
 
         InstructorStudentRecordsAjaxPageData data =
-                                        new InstructorStudentRecordsAjaxPageData(account, student, sessionToken, results);
+                                        new InstructorStudentRecordsAjaxPageData(account, student, sessionToken, results,
+                                                                                 sessionSubmissionStatusMap);
```

Comment:
This one, rather than squishing everything to the right, can look better like:
```java
InstructorStudentRecordsAjaxPageData data =
        new InstructorStudentRecordsAjaxPageData(account, student, sessionToken, results, sessionSubmissionStatusMap);
```